tasks/test.py

```diff
@@ -66,11 +70,16 @@ def test(ctx, targets=None, coverage=False, race=False, use_embedded_libs=False,
         matches = ["{}/...".format(t) for t in targets_list]
 
     for match in matches:
+        if invoke.platform.WINDOWS:
+            if match in windows_test_whitelist:
+                print("Skipping whitelisted directory {}\n".format(match))
+                continue
+
         coverprofile = ""
         if coverage:
             profile_tmp = "{}/profile.tmp".format(match)
             coverprofile = "-coverprofile={}".format(profile_tmp)
-        cmd = "go test -tags '{go_build_tags}' {race_opt} -short {covermode_opt} {coverprofile} {pkg_folder}"
+        cmd = 'go test -tags "{go_build_tags}" {race_opt} -short {covermode_opt} {coverprofile} {pkg_folder}'
```

Comment:
curious, did the other quote scheme break on windows?

Reply:
at least on windows, other quote scheme broke on execution:
`o test -tags 'jmx gce cpython zk process consul zlib snmp ec2 etcd apm docker' -race -short -covermode=atomic -coverprofile=./pkg\aggregator/profile.tmp ./pkg\aggregator?[0m
invalid flag argument for -tags: unterminated ' string`